SYCL/ESIMD/api/functional/value.hpp

```diff
@@ -112,6 +112,19 @@ template <typename DataT> struct value {
       return std::numeric_limits<DataT>::max();
     }
   }
+
+  static DataT ulp(DataT base_val, DataT direction) {
+    if constexpr (std::is_same_v<DataT, sycl::half>) {
+      return static_cast<sycl::half>(
+          (sycl::nextafter(base_val, direction) - base_val) * 8192);
+    } else {
+      return std::nextafter(base_val, direction) - base_val;
+    }
+  }
+
+  static DataT pos_ulp(DataT base_val) { return ulp(base_val, inf()); }
+
+  static DataT neg_ulp(DataT base_val) { return ulp(base_val, -inf()); }
```

Comment:
This func is unused. Shouldn't it be either deleted or used?

Reply:
This function will be useful in the next tests